indico/modules/rb_new/operations/rooms.py

```diff
@@ -50,7 +50,7 @@ def _filter_coordinates(query, filters):
 
 def _make_room_text_filter(text):
     text = '%{}%'.format(escape_like(text))
-    columns = ('name', 'site', 'division', 'building', 'floor', 'number', 'comments', 'full_name')
+    columns = ('full_name', 'site', 'division', 'building', 'floor', 'number', 'comments', 'full_name')
```

Comment:
ah we actually had `full_name` at the end of the list, so one of them can go away